integrations/tensorflow/e2e/iree_e2e_cartesian_product_test_suite.cmake

```diff
@@ -0,0 +1,276 @@
+
+# iree_py_test()
+#
+# Creates a test that executes a specified Python source file. The IREE Python
+# bindings are added to PYTHONPATH.
+#
+# Parameters:
+#   NAME: Name of the target
+#   SRC: Python source file to be executed.
+#   ARGS: Command line arguments to the Python source file.
+#   LABELS: Additional labels to apply to the test. The package path is added
+#       automatically.
+function(iree_py_test)
```

Comment:
How is this different from the main function we have of the same name?

Reply:
Whoops, didn't realize we did. I'll take a look if I can just use that.

Reply:
Switched to using that.